charts/oes/config/oes-sapor/application.yml

```diff
@@ -4,14 +4,19 @@ oes:
 spinnaker:
   authnEnabled: {{ .Values.sapor.config.spinnaker.authnEnabled }}
   baseurl: {{ .Values.sapor.config.spinnaker.spinGateURL }}
-  username: {{ .Values.sapor.config.spinnaker.spinuser }}
-  password: {{ .Values.sapor.config.spinnaker.spinpasswd }}
   externalBaseUrl: {{ .Values.sapor.config.spinnaker.spinExternalGateURL }}
-  adminLoginEnabled: {{ .Values.sapor.config.spinnaker.spinAdminLoginEnabled }}
-  {{- if .Values.sapor.config.spinnaker.spinAdminLoginEnabled }}
-  adminUsername: {{ .Values.sapor.config.spinnaker.spinAdminUsername }}
-  adminPassword: {{ .Values.sapor.config.spinnaker.spinAdminPassword }}
-  {{- end }}
+  ldap:
+    enabled: {{ .Values.sapor.config.spinnaker.ldap.ldapEnabled }}
+    username: {{ .Values.sapor.config.spinnaker.ldap.ldapUsername }}
+    password: {{ .Values.sapor.config.spinnaker.ldap.ldapPassword }}
+    adminLoginEnabled: {{ .Values.sapor.config.spinnaker.ldap.ldapAdminLoginEnabled }}
+    adminUsername:
```

Comment:
{{ .Values.sapor.config.spinnaker.spinAdminUsername }}

It should have this value

Reply:
Updated